womtool/src/main/scala/womtool/input/WomGraphMaker.scala

```diff
@@ -26,16 +26,8 @@ object WomGraphMaker {
     // Resolves for:
     // - Where we run from
     // - Where the file is
-    lazy val importResolvers: List[ImportResolver] = List(
-      DirectoryResolver(
-        DefaultPathBuilder.build(Paths.get(".")),
-        allowEscapingDirectory = false),
-      DirectoryResolver(
-        DefaultPathBuilder.build(Paths.get(mainFile.toAbsolutePath.toFile.getParent)),
-        allowEscapingDirectory = true
-      ),
-      HttpResolver()
-    )
+    lazy val importResolvers: List[ImportResolver] =
+      DirectoryResolver.localFilesystemResolvers(Some(mainFile)) :+ HttpResolver(relativeTo = None)
```

Comment:
Comment seems to be out of date - we don't resolve relative to where we run from anymore

I think the code is pretty self explanatory, so it could just go away

Reply:
Changed